api/envoy/config/metrics/v2/stats.proto

```diff
@@ -67,6 +67,10 @@ message StatsConfig {
   // Inclusion/exclusion matcher for stat name creation. If not provided, all stats are instantiated
   // as normal. Preventing the instantiation of certain families of stats can improve memory
   // performance for Envoys running especially large configs.
+  //
+  // .. warning::
+  //   Excluding stats may affect Envoy's behavior in undocumented ways. See
+  //   [issue-9856](https://github.com/envoyproxy/envoy/issues/9856) for more information.
```

Comment:
Can you add something to the effect of: "If abnormal behavior is observed please open an issue immediately."

Also, please check the rendered doc output either locally or in CI as sphinx uses a different external link format from markdown. Thank you!

Reply:
Will do. Also, since it seems like #8771 is the earlier issue and #9856 might get closed as a dup, I'll update the link in the docs to point to #8771